frontend/integration-tests-cypress/cypress/tests/project_crud.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { 
+  appHost, 
+  timeouts 
+} from '../support/utils';
+
+const getElemByDataId = dataId => cy.get(`[data-test-id="${dataId}"]`);
+
+// helpers other actions can use to build more complex actions
+const genericActions = {
+  clickActionMenuItemByText: (text) => {
+    getElemByDataId('actions-menu-button').click();
+    cy.get(`[data-test-action="${text}"]:not(.pf-m-disabled)`).click();
+  },
+}
+
+// namespaced sets of helpers, seem to need 3 things:
```

Comment:
The problem with this is that I don't know that `Cypress.Commands.add( ....)` supports the namespacing.

If it doesn't, I would propose something like this:

```ts
import * as project from './commands/project.ts';

Cypress.Commands.add('project.detailsPage.visit', project.detailsPage.visit);
Cypress.Commands.add('project.detailsPage.actions.deleteProject', project.detailsPage.actions.deleteProject);
Cypress.Commands.add('project.detailsPage.assert.titleShouldBe', project.detailsPage.assert.titleShouldBe);

Cypress.Commands.add('pod.detailsPage.blah.blah, pod.blah.blah);
```
Need to play with that a bit and make sure its not a hassle to type and use.